src/bayesian_inference/mcmc.py

```diff
@@ -59,23 +59,22 @@ def run_mcmc(config, closure_index=-1):
     )
     emulation_results = emulation_config.read_all_emulator_groups()
 
-    # FIXME: The experimental results are NOT in the same order as the emulator groups!
     # Load experimental data into arrays: experimental_results['y'/'y_err'] (n_features,)
     # In the case of a closure test, we use the pseudodata from the validation design point
-    experimental_results = data_IO.data_array_from_h5(config.output_dir, 'observables.h5', pseudodata_index=closure_index)
+    experimental_results = data_IO.data_array_from_h5(config.output_dir, 'observables.h5', pseudodata_index=closure_index, observable_filter=emulation_config.observable_filter)
 
     # TODO: By default the chain will be stored in memory as a numpy array
     #       If needed we can create a h5py dataset for compression/chunking
 
     # We can use multiprocessing in emcee to parallelize the independent walkers
-    with Pool() as pool:
+    with Pool(processes=1) as pool:
```

Comment:
Is there a particular reason for this change? (We still have a lot of optimizing to do for MCMC so totally fine if this is needed for now...but just wanted to check)

Reply:
No - I just used it to simplify debugging and was overzealous with my commit, sorry!

Reply:
Will fix in the morning

Reply:
Ah ok :) No worries, thanks! And no rush.

Reply:
This is now fixed - merging